seva-launcher/seva-compose.go

```diff
@@ -45,11 +45,17 @@ func start_app(command WebSocketCommand) WebSocketCommand {
 	return command
 }
 
-func web_proxy(command WebSocketCommand) WebSocketCommand {
+func save_settings(command WebSocketCommand) WebSocketCommand {
 	log.Println("I'm in")
 	
 	// Add "sysconfig/docker file" updation code here
-    
+    err := ioutil.WriteFile("/etc/sysconfig/docker", []byte(command.Command), 0666)
```

Comment:
You need to have a check in place to switch between the systemd override and the sysconfig override. Check to see if `/etc/systemd/system` exists.

Also, I'm sure you're aware that the command needs to be deserialized and repackaged into the correct config.